pkg/server/skydb/pq/listener.go

```diff
@@ -122,7 +122,7 @@ func (l *recordListener) Listen() {
 				log.WithFields(logrus.Fields{
 					"pqNotification": pqNotification,
 					"err":            err,
-				}).Errorln("pq/listener: failed to fetch notification")
+				}).Warnln("pq/listener: failed to fetch notification")
```

Comment:
Actually we should be using the non-ln version of the logging call. Personally I use `{Debug,Info,Warn,Error}f` which provides the same formatting API as `fmt.Printf`.